data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report3_col_suppression.py

```diff
@@ -130,46 +118,35 @@
 
 FROM `{{project_id}}.{{deid_cdr}}.measurement`
 """)
-q = query.render(project_id=project_id, deid_cdr=deid_cdr)
-df1 = execute(client, q)
-if df1.loc[0].sum() == 0:
-    df = df.append({
-        'query': 'Query3 Measurement',
-        'result': 'PASS'
-    },
-                   ignore_index=True)
+q = query.render(project_id=project_id,deid_cdr=deid_cdr)  
+df1=execute(client, q) 
+if df1.loc[0].sum()==0:
+ df = df.append({'query' : 'Query3 Measurement', 'result' : 'PASS'},  
+                ignore_index = True) 
 else:
-    df = df.append({
-        'query': 'Query3 Measurement',
-        'result': 'Failure'
-    },
-                   ignore_index=True)
-df1.T
+ df = df.append({'query' : 'Query3 Measurement', 'result' :  'Failure'},  
+                ignore_index = True) 
+df1.T  
 
-# # 4 Verify the following columns in the AOU_DEATH table have been set to null:
+# # 4 Verify the following columns in the DEATH table have been set to null:
 
```

Comment:
I don't think changes to other queries are needed. This seems to be an old version of the notebook.

Reply:
No longer an issue.